src/officehours_api/backends/zoom.py

```diff
@@ -110,10 +110,21 @@ def _spend_authorization_code(cls, code: str, request) -> ZoomAccessToken:
         logger.debug("Received authorization code from Zoom")
         return resp.json()
 
+    @classmethod
+    def _clear_backend_metadata(cls, user: User) -> None:
+        user.profile.backend_metadata['zoom'] = {}
+        user.profile.save()
+        logger.info(f'Removed Zoom metadata for user {user.id} to prompt re-authorization.')
+
+    @classmethod
+    def _calculate_expires_at(cls, expires_in: int) -> float:
+        return time() + expires_in - cls.expiry_buffer_seconds
+
     @classmethod
     def _get_access_token(cls, user: User) -> str:
-        zoom_meta = user.profile.backend_metadata['zoom']
+        zoom_meta = user.profile.backend_metadata['zoom'].copy()
```

Comment:
Are you doing this just to better protect this from backend_metadata changes? I don't see `zoom_meta` actually modified anywhere in this method any longer and the only two uses of it is before `user.profile.backend_metadata` is modified. 

It could just be removed and replaced in those two cases with `user.profile.backend_metadata['zoom']`? I don't really see any preferences for preferred style patterns out there in Python.

Reply:
The `copy` may not longer be necessary. @jlost raised a concern in DM that the reference to `zoom_meta` and subsequent updating could unintentionally persist values that we meant to clear using `_clear_backend_metadata`. I don't think it was likely to ever come up, because of the exception throwing, but I think we wanted to do some future proofing and stay away from this kind of possibility. I think I've avoided that by not updating `zoom_meta`, so maybe the copy can be removed, but I was doing it out of an abundance of caution.

Reply:
Since you're no longer mutating `zoom_meta`, I think it would be OK to remove the `.copy()`. It doesn't cause any harm though.

Reply:
I removed `.copy()`, thanks!